pom.xml

```diff
@@ -115,6 +115,11 @@ SOFTWARE.
       <artifactId>asto</artifactId>
       <version>0.23.1</version>
     </dependency>
+    <dependency>
```

Comment:
The library is synchronous, it would not work well. You can make HTTP requests to Github with Vert.x HTTP client.

Reply:
@Sammers21 it doesn't make any sense here: authentication is a blocking operation in any case, we can't continue processing the request without performing synchronous call and authenticating the user.

Reply:
@g4s8, it is a blocking process for a user, not necessary for the application.

Reply:
@Sammers21 I don't see any benefits using reactive client here: we still need to wait for response in a blocking way, so the performance of this call will be the same.

Reply:
@g4s8, i guess that is because interface design is bad. The user method should be returning `Maybe<String>` or `Single<Optional<String>>` or `CompletableFuture<Optional<String>>`.